modules/osdk-apiservices.adoc

```diff
@@ -5,67 +5,52 @@
 
-.Owned APIService fields
+.Owned API service fields
 [cols="2a,5a,2",options="header"]
 |===
-|Field |Description |Required/Optional
+|Field |Description |Required/optional
 
 |`Group`
-|Group that the APIService provides, for example `database.example.com`.
+|Group that the API service provides, for example `database.example.com`.
 |Required
 
 |`Version`
-|Version of the APIService, for example `v1alpha1`.
+|Version of the API service, for example `v1alpha1`.
 |Required
 
 |`Kind`
-|A kind that the APIService is expected to provide.
+|A kind that the API service is expected to provide.
 |Required
 
 |`Name`
-|The plural name for the APIService provided
+|The plural name for the API service provided
 |Required
 
 |`DeploymentName`
-|Name of the deployment defined by your CSV that corresponds to your APIService
-(required for owned APIServices). During the CSV pending phase, the OLM Operator
-searches the InstallStrategy of your CSV for a Deployment spec with a matching
-name, and if not found, does not transition the CSV to the install ready phase.
+|Name of the deployment defined by your CSV that corresponds to your API service (required for owned API services). During the CSV pending phase, the OLM Operator searches the `InstallStrategy` of your CSV for a Deployment spec with a matching name, and if not found, does not transition the CSV to the "install ready" phase.
 |Required
 
 |`DisplayName`
-|A human readable version of your APIService name, for example `MongoDB Standalone`.
+|A human readable version of your API service name, for example `MongoDB Standalone`.
 |Required
 
 |`Description`
-|A short description of how this APIService is used by the Operator or a
-description of the functionality provided by the APIService.
+|A short description of how this API service is used by the Operator or a description of the functionality provided by the API service.
 |Required
 
 |`Resources`
-a|Your APIServices own one or more types of Kubernetes objects. These are listed
-in the resources section to inform your users of the objects they might need to
-troubleshoot or how to connect to the application, such as the Service or
-Ingress rule that exposes a database.
-
-It is recommended to only list out the objects that are important to a human,
-not an exhaustive list of everything you orchestrate. For example, ConfigMaps
-that store internal state that should not be modified by a user should not
-appear here.
+a|Your API services own one or more types of Kubernetes objects. These are listed in the resources section to inform your users of the objects they might need to troubleshoot or how to connect to the application, such as the service or ingress rule that exposes a database.
+
+It is recommended to only list out the objects that are important to a human, not an exhaustive list of everything you orchestrate. For example, config maps that store internal state that should not be modified by a user should not appear here.
```

Comment:
>should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.
For example, use “Reset the parameters,” not “You should reset the parameters.”